babel/dates.py

```diff
@@ -243,18 +243,17 @@ def get_timezone(zone: str | datetime.tzinfo | None = None) -> datetime.tzinfo:
     if not isinstance(zone, str):
         return zone
 
-    exc = None
     if pytz:
         try:
             return pytz.timezone(zone)
-        except pytz.UnknownTimeZoneError as exc:  # noqa: F841
-            pass
+        except pytz.UnknownTimeZoneError as e:
+            exc = e
```

Comment:
If the variable is not used, can't it be removed?
```suggestion
        except pytz.UnknownTimeZoneError:
            pass
```